yarn.lock

```diff
@@ -1,8736 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Are we keeping yarn.lock or not? I've had both understandings over the last week.